docs/developer/develop-minikube.md

```diff
@@ -0,0 +1,178 @@
+VSCode is an IDE that combined with the Google Cloud Code extension will manage the dependencies needed for the development workflow and provide an interface to debug through breakpoints and watches.
+
+---
+## Setup
+### Requirements
+Please install the following requirements on your system.
+* [Docker](https://docs.docker.com/engine/install/), you can install Docker Desktop or only the engine.
+   * Make sure docker can be execute as non-root (`docker run hello-world`). See [here](https://docs.docker.com/engine/install/linux-postinstall/) for instructions.
+   * Some operating systems can run Minikube (faster) with other hypervisors. If you choose so expect some extra custom setup to be done. We advice the use of Docker for Minikube.
+* [Helm](https://helm.sh/docs/intro/install/), to manage the deployment of Turbinia.
+* [VSCode](https://code.visualstudio.com/Download), our IDE of choice for this setup.
+
+### Configure VSCode
+Start VSCode and install the following extensions:
+* [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+* [Gemini Code Assist + Google Cloud Code](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.cloudcode)
+
+Restart VSCode. Once VSCode has restarted let's install the development dependencies (minikube, skaffold and kubectl). We will let the Google Cloud Code extension manage the dependencies and they will be installed in a seperate folder, not in your system folders. In VSCode:
+ * Control-Shift-P (or Command-Shift-P on Macs) to open the command menu
+ * Choose `Cloud Code:  Control Minikube`
+ * Choose `Continue with Managed dependencies`
+
+This will have VSCode install the dependencies (minikube, skaffold and kubectl) and it can take a while.
+
+![Cloud Code Dependencies Install](../images/cloudcode-dependencies.png)
+
+### Start Minkube cluster
+We will now start the minikube k8s cluster
+ * Control-Shift-P (or Command-Shift-P on Macs) to open the command menu
+ * Choose `Cloud Code:  Control Minikube`
+ * Choose `Start` 
+
+![Minikube Start](../images/cloudcode-startminikube.png)
+
+![Minikube Start Output](../images/cloudcode-minikubestart.png)
+
+NOTE: If you want to change the default cluster CPU and Memory usage, you can set those before starting the cluster
+
+    $ minikube config set cpus 4
+    $ minikube config set memory 16384
+
+### Turbinia source and deployment code 
+Now we have VSCode setup we are going to get a copy of the Turbinia source and deployment code.
+Clone the [Turbinia repository](https://github.com/google/turbinia) by forking the Turbinia repository into your own Github account and clone it locally from there.
+
+    $ git clone ssh://git@github.com:[YOUR_GITHUB_ACCOUNT]/turbinia.git
+
+Let's get the helm charts for the Turbinia deployment. In your cloned turbinia repository
+
+    $ mkdir charts && cd charts
+
+    $ helm pull oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia --untar && cd ..
```

Comment:
Update to 
```
helm repo add osdfir-charts https://google.github.io/osdfir-infrastructure
helm pull osdfir-charts/turbinia --untar && cd ..
```

Since charts are no longer being updated in the OCI registry

Reply:
Done